### PR TITLE
Handle 'you are not a member of this team' error

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

File: shared/actions/teams/index.js [hunk 1]
```diff
@@ -283,16 +283,42 @@ const _createNewTeamFromConversation = function*(
   }
 }
 
+const isYouAreNotMemberError = (e: Error) => {
+  // TODO: Can we use a constant here instead?
+  const youAreNotMemberErrorCode = 2711
```
Comment (akalin-keybase) [inline shared/actions/teams/index.js]: This comes from the server, so I don't think there's an existing constant for it.

Comment (chrisnojima) [reply shared/actions/teams/index.js]: can put it into the avdl or at least put it into constants/teams

**Comment by akalin-keybase** [comment]
Per discussion on CORE-6647, not needed anymore.

**Pull request closed without merging**